internal/storage/types/location_config.go

```diff
@@ -143,6 +159,25 @@ func (c URLConfig) OpenBucket(ctx context.Context) (*blob.Bucket, error) {
 	if err != nil {
 		return nil, fmt.Errorf("open bucket by URL: %v", err)
 	}
+	return b, nil
+}
+
+type SSConfig struct {
+	SS string `json:"url"`
+}
 
+func (c SSConfig) Valid() bool {
+	return c.SS != ""
+}
+
+func (c SSConfig) OpenBucket(ctx context.Context) (*blob.Bucket, error) {
+	opts := ssblob.Options{
+		Context: ctx,
```

Comment:
We don't need to share this context with ssblob.Options, just ignore it like we did in SFTPConfig. It's only useful when creating a bucket targeting blobstores like like Google Cloud Storage where the init process needs to deal with some authentication concerns that may require extra network connections.